src/main/java/com/moodybluez/enterprise/controller/EntryController.java

```diff
@@ -49,7 +48,9 @@ Entry getByMonth(@PathVariable Integer year, @PathVariable Integer month, @PathV
         List<Entry> entities = entryService.fetchByMood(id);
 
         for(Entry entry:entities){
-            int weekday = entry.getDate().getDay();
+            Calendar calendarInstance = Calendar.getInstance();
+            calendarInstance.setTime(entry.getDate());
+            int weekday = calendarInstance.get(Calendar.DAY_OF_WEEK);
```

Comment:
Ditto as previous - "Technical Debt - The method being used here "getDay()" has been deprecated for a long time and this could really be a problem, especially if JVM is running newer version which will require you to update this anyway, thus creating technical debt."